doc/getting_started/index.rst

```diff
@@ -3,319 +3,409 @@
+            ./zephyr-sdk-0.10.3-setup.run -- -d ~/zephyr-sdk-0.10.3
 
-Zephyr applications are built to run on specific hardware, which is
-called a "board"\ [#board_misnomer]_. We'll use the :ref:`reel_board
-<reel_board>` here, but you can change ``reel_board`` to another value if you
-have a different board. See :ref:`boards` or run ``west boards`` from anywhere
-inside the ``zephyrproject`` directory for a list of supported boards.
+      #. Set environment variables:
 
-#. Go to the zephyr repository:
+         .. code-block:: bash
 
-   .. code-block:: console
+            export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
+            export ZEPHYR_SDK_INSTALL_DIR=~/zephyr-sdk-0.10.3
 
-      cd zephyrproject/zephyr
+   .. group-tab:: macOS
 
-#. Set up your build environment:
+      The Zephyr SDK is not supported on macOS.  See instructions for
+      :ref:`installing 3rd-party toolchains<gs_toolchain>`.
 
-   .. code-block:: console
+   .. group-tab:: Windows
 
-      # Linux and macOS
-      source zephyr-env.sh
+      The Zephyr SDK is not supported on Windows.  See instructions for
+      :ref:`installing 3rd-party toolchains<gs_toolchain>`.
+
+.. _getting_started_run_sample:
+
+.. rst-class:: numbered-step
+
+Build the Blinky Application
+****************************
 
-      # Windows
-      zephyr-env.cmd
+The sample :ref:`blinky-sample` blinks an LED on the target board.  By
+building and running it, we can verify that the environment and tools
+are properly set up for Zephyr development.
 
-#. Build the Hello World sample for the ``reel_board``:
+#. Set build environment variables:
 
-   .. zephyr-app-commands::
-      :app: samples/hello_world
-      :board: reel_board
-      :goals: build
+   .. tabs::
 
-The main build products will be in :file:`build/zephyr`;
-:file:`build/zephyr/zephyr.elf` is the Hello World application binary in ELF
-format. Other binary formats, disassembly, and map files may be present
-depending on your board.
+      .. group-tab:: Ubuntu
 
-The other sample applications in :zephyr_file:`samples` are documented in
-:ref:`samples-and-demos`. If you want to re-use an existing build directory for
-another board or application, you need to pass ``-p=auto`` to ``west build``.
+         .. code-block:: bash
 
-Run the Application by Flashing to a Board
-==========================================
+            cd ~/zephyrproject/zephyr
+            source zephyr-env.sh
 
-Most "real hardware" boards supported by Zephyr can be flashed by running
-``west flash``. However, this may require board-specific tool installation and
-configuration to work properly.
+      .. group-tab:: macOS
 
-See :ref:`application_run` in the Application Development Primer and your
-board's documentation in :ref:`boards` for additional details.
+         .. code-block:: bash
 
-Run the Application in QEMU
-===========================
+            cd ~/zephyrproject/zephyr
+            source zephyr-env.sh
```

Comment:
I'd still like to eliminate explicit use of environment variables in the command line flow using west someday.

Would you be open to removing these scripts from the GSG at that time?

Reply:
Yes, if we can assume that EVERYONE will be using west :)

Reply:
> Yes, if we can assume that EVERYONE will be using west :)

We can keep environment variable support around as an advanced topic or for people who don't use west, just talking about the GSG and 'recommended easiest' workflows here.